README.md

```diff
@@ -67,30 +95,87 @@ npx mikro-orm migration:up
 npm run start
 ```
 
-## 7. Install the SSL certificate
+## 5. Install the SSL certificate
 
 `docker compose cp caddy:/data/caddy/pki/authorities/local/root.crt $LOCAL_PATH`
 
-You can then install in your OS (at least on Mac it's a double click).
-However, most browsers have their own cert management.
+You can then install in your OS (at least on Mac it's a double click). However, most browsers have their own cert management.
 
 On Chrome, you can add the cert via chrome://certificate-manager/.
 
-## 8. Access the running services
+## 6. Access the running services
 
 - Frontend: https://grassroots.org
 - Backend API: https://grassroots.org/api
 
+# Common Development Commands
+
+Using the `./run` script for everyday development:
+
+```bash
+# Backend development
+./run "cd grassroots-backend && npm install"
+./run "cd grassroots-backend && npm run start:dev"
+./run "cd grassroots-backend && npm test"
+
+# Frontend development
+./run "cd grassroots-frontend && npm install"
+./run "cd grassroots-frontend && npm run dev"
+./run "cd grassroots-frontend && npm run build"
+
+# Database operations
+./run "cd grassroots-backend && npx mikro-orm migration:up"
+./run "cd grassroots-backend && npx mikro-orm migration:create"
+
+# General commands
+./run npm --version
+./run git status
+```
+
+# Git Hooks Integration
```

Comment:
Folks shouldn't need to worry about this, it should Just Work. Can we remove this section?